### ISSUE TITLE
ENH: Allow tool to run in a web browser

### ISSUE THREAD

**Issue by corneliusroemer** [lenaschimmel/sarscov2recombinants#13]
It'd be super cool if the tool ran in a web browser.

Drop (aligned) sequences and see it displayed.

This may be not worth the probably not quite insignificant effort though.

It would reduce barrier to entry significantly.

**Comment by FedeGueli** [comment]
Agree vm!

**Comment by lenaschimmel** [comment]
Well, I know absolutely nothing about actually running python _in_ the browser, but it should be possible to run the tool on a server and provide a web front end.

When I searched for it, I stumbled across [cmd2web](https://github.com/ryanlayer/cmd2web). (Coincidentally, it's from @ryanlayer who's also doing genetics.) I did not yet have time to take a closer look or even try it out.

With a web-based tool, there'd be a lot of interesting applications, e.g. linking from other tools to this one ("click here to see your data in sc2r") or providing on-the-fly conversion like I did for csv files with [csv2svg4md](https://github.com/lenaschimmel/csv2svg4md).

**Comment by corneliusroemer** [comment]
Yes exactly, sorry, I didn't mean to run it in a browser like Nextclade does with web assembly - that'd be complicated.

With a server would be perfect. Actually it may be quite easy for you given your experience (for me it wouldn't :D )

On a server you could align with Nextalign for example, if fasta is not yet aligned.

**Comment by lenaschimmel** [comment]
Yesterday @zach-hensel pointed out that sc2rf can be run from a web browser through [Google Colab](https://colab.research.google.com/), which is a modified version of [Jupyter-Notebooks](https://www.jupyter.org/).

This approach can work quite nicely, as shown by [ColabFold](https://github.com/sokrypton/ColabFold) (also recommended by Zach).

I will try to set up a Collab Notebook soon and share the link here and in the README.
